openassessment/xblock/test/test_studio.py

```diff
@@ -106,7 +107,8 @@ class StudioViewTest(XBlockHandlerTestCase):
         "example-based-assessment": "oa_ai_assessment_editor",
         "peer-assessment": "oa_peer_assessment_editor",
         "self-assessment": "oa_self_assessment_editor",
-        "student-training": "oa_student_training_editor"
+        "student-training": "oa_student_training_editor",
+        "staff-assessment": "oa_staff_assessment_editor",
```

Comment:
Turns out, adding staff to this line and `test_render_editor_assessment_order` below will cause a failure before this fix due to the bug this ticket was filed against. I've removed the test I initially added since it is now superfluous.